tools/scripts/llm/README.md

```diff
@@ -0,0 +1,21 @@
+# CLDR LLM Validator
+
++---------------------------------------------------+
+| GSoC Contributor: preetsureshbhai.sojitra@email.ucr.edu |
+| Mentors:                                           |
+| - younies@unicode.org                              |
+| - annemarie@unicode.org                            |
+| Tech Lead: mark@unicode.org                        |
+| Doc Status: Draft                                  |
+| Last Update: June 11, 2025                          |
+| Project Information:                               |
+| This project is part of GSoC initiation:           |
+| [Unicode Inc. Project Details](https://summerofcode.withgoogle.com/organizations/unicode-inc/projects/details/lE93K0ho) |
++---------------------------------------------------+
```

Comment:
doesn't render quite right in github

Reply:
could probably link to github profiles vs email such as https://github.com/aeapple